Packs/CrowdStrikeFalcon/ReleaseNotes/2_0_28.md

```diff
@@ -0,0 +1,75 @@
+
+#### Classifiers
+
+##### CrowdStrike Falcon Incident Classifier
+
+- Added new incident type "Crowdstrike Command and Scripting"
+
+#### Incident Fields
+
+##### New: Grand Parent Process
+
+- New: Grand Parent Process <~XSOAR> (Available from Cortex XSOAR 6.10.0).</~XSOAR>
+
+#### Incident Types
+
+##### New: Crowdstrike Command And Scripting
+
+- New: Handles mitre T1059 CS alerts<~XSOAR> (Available from Cortex XSOAR 6.10.0).</~XSOAR>
```

Comment:
MITRE not mitre, CrowdStrike not CS.
Handles MITRE T1059 - Command And Scripting CrowdStrike alerts